### DISCUSSION TITLE
which is better? WebCodecs or TinyH264

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#478] [Q&A] [answered]
the points that I am asking about are

- quality (color depth/accuracy and resolution)
- speed on lower-end devices

**Comment by yume-chan** [comment, marked as answer]
Refer to their README files for comparison:

https://github.com/yume-chan/ya-webadb/tree/main/libraries/scrcpy-decoder-webcodecs#readme
https://github.com/yume-chan/ya-webadb/tree/main/libraries/scrcpy-decoder-tinyh264#readme

Decoding H.264 is a deterministic algorithm so I don't think they will differ on quality. However, WebCodecs does support higher profile/level so better encoding quality at same bit rate if those configurations are also supported by the encoder on device.
